setup.cfg

```diff
@@ -5,6 +5,8 @@ testpaths = tests
 markers =
     ignore_article: Configure the settings DJANGO_FSM_LOG_IGNORED_MODELS to ignore
                     Article Model.
+    ignore_article_integer: Configure the settings DJANGO_FSM_LOG_IGNORED_MODELS to ignore
```

Comment:
What are you trying to achieve with this marker ?

Reply:
I have seen that the other model uses a marker. I simply copied it. Should I remove it?

Reply:
Usually markers are used to select a subset of tests or to trigger side effects on others.
You don't do any of this here, so yes you can remove it.